lib/services/pbxproj-dom-xcode.ts

```diff
@@ -0,0 +1,7 @@
+import * as PbxprojDomXcodeModule from "pbxproj-dom/xcode";
+
+declare global {
+	type IPbxprojDomXcode = typeof PbxprojDomXcodeModule;
+}
+
+$injector.register("pbxprojDomXcode", () => require("pbxproj-dom/xcode"), true);
```

Comment:
there's no need to pass `true` here - in case the value is undefined, `true` is set as default:
https://github.com/telerik/mobile-cli-lib/blob/master/yok.ts#L284